tests/phpunit/tests/theme/wpThemeJsonResolver.php

```diff
@@ -466,6 +466,23 @@ public function test_add_theme_supports_are_loaded_for_themes_without_theme_json
 		$this->assertSame( $color_palette, $settings['color']['palette']['theme'] );
 	}
 
+	/**
+	 * Tests that classic themes still get core default settings such as color palette and duotone.
+	 *
+	 * @ticket 60136
+	 */
+	public function test_core_default_settings_are_loaded_for_themes_without_theme_json() {
+		switch_theme( 'default' );
+		remove_theme_support( 'editor-gradient-presets' );
```

Comment:
I have added this here as a temp fix for the same problem encountered in #5816. If that patch is committed first, we should be able to safely remove this.